defaults/main.yml

```diff
@@ -14,6 +14,7 @@ LOAD_BALANCER_CERTS_DIR: /etc/haproxy/certs
 LOAD_BALANCER_CONF_DIR: /etc/haproxy/conf.d
 LOAD_BALANCER_BACKENDS_DIR: /etc/haproxy/backends
 LETSENCRYPT_ARCHIVE_DIR: /etc/letsencrypt/archive
+LETSENCRYPT_LIVE_DIR: /etc/letsencrypt/live
```

Comment:
Could you please use the new `LETSENCRYPT_LIVE_DIR` variable in these files too?
* [`templates/haproxy-config-watcher`](https://github.com/open-craft/ansible-load-balancer/blob/82c10ba6d222ac79fce70a240a4ce7fc535cdf66/templates/haproxy-config-watcher#L17)
* [`templates/manage_certs.py`](https://github.com/open-craft/ansible-load-balancer/blob/44f54d136f329936ead49a63f1f514bb9dcce506/templates/manage_certs.py#L197)
* [`integration/test.yml`](https://github.com/open-craft/ansible-load-balancer/blob/b729113c24d877aae44246ed4888eb73f773bd19/tests/integration/test.yml#L34)